src/components/header/header.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, HStack, VStack } from "@nypl/design-system-react-components";
+import React, { useEffect, useState } from "react";
+import DCLogo from "../logo/logo";
+
+const Header = () => {
+  const [isScrolled, setIsScrolled] = useState(true);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.scrollY > 100) {
+        setIsScrolled(false);
+      } else {
+        setIsScrolled(true);
+      }
+    };
+
+    window.addEventListener("scroll", handleScroll);
+
+    return () => {
+      window.removeEventListener("scroll", handleScroll);
+    };
+  }, []);
```

Comment:
Lines 6-22 can be a custom hook. It's not necessary because I don't think you will need to implement this in any other component. But, it _could_ be reusable in the future and added to the DS so other apps can implement a consistent sticky behavior. This also depends on the `100` number and how/why it was chosen.